packages/block-library/src/cover/style.scss

```
@@ -167,14 +167,19 @@
 			width: auto;
 		}
 	}
-}
 
-.wp-block-cover__image-background,
-.wp-block-cover__video-background {
-	position: absolute;
-	width: 100%;
-	height: 100%;
-	object-fit: cover;
+	// Extra specificity for in edit mode where other styles would override it otherwise.
+	img.wp-block-cover__image-background,
+	video.wp-block-cover__video-background {
```

Do we need the `img` and `video` element-based selectors? :sweat: I was hoping that moving the selectors into `.wp-block-cover` would be specific enough.

```suggestion
	.wp-block-cover__image-background,
	.wp-block-cover__video-background {
```

---

It wasn't working for me without it, unfortunately. Without it, the specificity is the same and it still gets overridden.

---

Since the extra specificity is only needed in the editor, I could duplicate these styles in edit.scss with the extra specificity and move these out to where they were before. But, I figured it would be better to avoid duplication between edit and save wherever possible.